Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,11 @@
 import React from 'react';
 import './Header.css';
+
```

Comment:
I noticed you didn't include the time stamp in your header title. You can use Moment.js and just install, import it and include the desired format in your code.